assets/javascripts/admin_needle.js

```diff
@@ -196,4 +208,31 @@ function setupAdminNeedles() {
 
     $('#last_seen_filter').change(reloadNeedlesTable);
     $('#last_match_filter').change(reloadNeedlesTable);
+    $('#btn_custom_last_seen').click(reloadNeedlesTable);
+    $('#btn_custom_last_match').click(reloadNeedlesTable);
```

Comment:
When I remember jQuery correctly it is possible to write this as `$('#last_seen_filter, #last_match_filter, #btn_custom_last_seen, #btn_custom_last_match').click(reloadNeedlesTable)`. The same counts for the other jQuery selectors. Note that it would also be possible to assign a class to these elements and just use that.